src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -192,7 +192,12 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
                     </MSKTab>
                 },
                 hide:()=>{
-                    return this.resultsViewPageStore.hugoGeneSymbols.length < 2;
+                    if (!this.resultsViewPageStore.isSampleAlteredMap.isComplete) {
```

Comment:
i think an "or statement" might make this more clear.  also put a comment here.   we are using the size of isSampleAlteredMap as a proxy for the number of things we have to compare.   that's business logic we should make that more clear.